dockerfile_functions.sh

```diff
@@ -651,6 +681,13 @@ RUN set -eux; \\
     case "\${ARCH}" in \\
 EOI
 	print_java_install_pre "${file}" "${pkg}" "${bld}" "${btype}" "${osfamily}" "${os}"
+	if [ "${btype}" == "slim" ]; then
+		if [ ${os} == "ubi" ]; then	
```

Comment:
Almost there ! `${os}` needs to be in quotes.

Reply:
Good catch, thank you